RealmSwift/List.swift

```diff
@@ -834,3 +834,10 @@ extension List: PropertyNameConvertible {
         return (key: rlmArray.propertyKey, isLegacy: rlmArray.isLegacyProperty)
     }
 }
+
+// MARK: Thread Safe Wrapper
+
+extension List: ThreadSafeWrappable {
+    /// :nodoc:
+    public static var confinedType: PropertyType { Element._rlmType }
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Is it okay to have an empty protocol?

Reply:
Results will need to be excluded too.

Reply:
Yes.